sevenseconds/config/kms.py

```diff
@@ -14,9 +14,9 @@ def configure_kms_keys(account: object, region):
             continue
         key = json.loads(json.dumps(key_config).replace('{account_id}', account.id))
         with ActionOnExit('Searching for key "{}"..'.format(key_alias)) as act:
-            exist_aliases = kms_client.list_aliases()
+            exist_aliases = paginate(kms_client.list_aliases)
             found = False
-            for alias in exist_aliases['Aliases']:
+            for alias in exist_aliases:
```

Comment:
`kms.describe_key(KeyId="alias/{}".format(key_alias))` instead? This way you don't need pagination at all, and it's much more efficient.

Reply:
Docs [here](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/kms.html#KMS.Client.describe_key).

Reply:
Good idea. I have changed it!